client/src/enums/BoardTableColumns.ts

```diff
@@ -5,5 +5,5 @@ export enum BoardTableColumns {
   CREATED_AT = "Created at",
```

Comment:
![image](https://github.com/user-attachments/assets/2723fa1c-1f7f-43fe-8d4f-a3e095d86414)

I think a phase `Created at 5 days ago` is incorrect

Shouldnt be either `Created at [specific date]` or just `Created 5 days ago`? Same goes with Modified

Reply:
Okay done